dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -30,8 +30,27 @@ FROM {{ .from }}
 {{- if contains .from "ubi-minimal" }}
 RUN for iter in {1..10}; do microdnf update -y && microdnf install -y shadow-utils jq &&  microdnf clean all && exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
 {{- else }}
+
+RUN set -e ; \
+  case "$(arch)" in \
+    aarch64) \
+        for iter in {1..10}; do yum update --setopt=tsflags=nodocs -y \
+            && yum install --setopt=tsflags=nodocs -y http://mirror.centos.org/centos/7/updates/aarch64/Packages/bind-license-9.11.4-26.P2.el7_9.5.noarch.rpm \
```

Comment:
When this is merged please create a follow-up issue to clean this.

Reply:
Sure, I hope to remove this terrible hack as soon as possible.